### PR TITLE
[FIX][AGW] Fixing the health checking script reported with #6379

### PR DESCRIPTION
**Pull request by jordanvrtanoski**
[lte][agw] Fixing the heath checking script 

## Summary

Health checking script was showing error and failed to display the status of the services

## Test Plan

n/a

## Additional Information

- [ ] This change is backwards-breaking

This problem is reported on v1.5

### CODE REVIEW

File: orc8r/gateway/python/magma/common/health/health_service.py [hunk 1]
```diff
@@ -63,7 +63,7 @@ def ping_status(self, host):
     def get_error_summary(self, service_names):
         configs = {service_name: load_service_mconfig_as_json(service_name)
                    for service_name in service_names}
-        res = {service_name: Errors(log_level=configs[service_name]['logLevel'],
+        res = {service_name: Errors(log_level=configs[service_name]['logLevel']  if 'logLevel' in configs[service_name] else "INFO",
```
Comment (pshelar) [inline orc8r/gateway/python/magma/common/health/health_service.py]: can you fix the style related issues.

**Comment by pshelar** [comment]
There is also python linter issue reported in lte-test that needs fixing.

File: orc8r/gateway/python/magma/common/health/health_service.py (hunk 1 above)
Comment (jordanvrtanoski) [reply orc8r/gateway/python/magma/common/health/health_service.py]: fixed

**Comment by themarwhal** [comment]
Hi @jordanvrtanoski ! Here are some more information on the python lint errors / warning.

To run the pylint test...
```
# in magma VM
cd magma/lte/gateway
make test_python

# ... OR to only run the pylint test
cd magma/lte/gateway
make build_python # only need to run this once
cd magma/orc8r/gateway
. /home/vagrant/build/python/bin/activate; /home/vagrant/build/python/bin/nosetests -s magma/health
```

To run `flake8` locally (these are the annotations you see in the PR)
```
# install docker if you don't have it already
# on any machine that has docker
cd magma/lte/gateway/python
./precommit --build # build docker image
./precommit --lint -p PATH1 PATH2
```

**Comment by jordanvrtanoski** [comment]
Issues addressed.

**Review by pshelar — APPROVED** [review]
Thanks for fixing all issues.

**Review by themarwhal — APPROVED** [review]

**Comment by themarwhal** [comment]
Hey @jordanvrtanoski, is this ready to be merged? If it is, I will merge.

**Comment by jordanvrtanoski** [comment]
No further changes from my side. It's ready for merge.

**Pull request merged**